indicators/utils.py

```diff
@@ -0,0 +1,25 @@
+
+def populate_ids_for_indicator(apps, schema_editor):
+    """
+    Populate primary keys values for migration in CountryIndicator model
+    """
+    model = apps.get_model('indicators', 'CountryIndicator')
+    for field_id, field in enumerate(model.objects.all()):
+        field.id = field_id + 1
+        field.save()
+
+
+def opposite_letter_case(data_string):
```

Comment:
I guess it might be reasonable to move definition of `opposite_letter_case` somewhere into `tests` package, just because it's used only there.